tModPorter/tModPorter/Config.ModLoader.cs

```diff
@@ -500,6 +500,7 @@ public static partial class Config
 		HookRemoved("Terraria.ModLoader.ModTile", "Drop", "Use CanDrop to decide if an item should drop. Use GetItemDrops to decide which item drops. Item drops based on placeStyle are handled automatically now, so this method might be able to be removed altogether.");
 		RenameInstanceField("Terraria.ModLoader.ModTile", from: "ChestDrop", to: "ItemDrop");
 		RenameInstanceField("Terraria.ModLoader.ModTile", from: "DresserDrop", to: "ItemDrop");
+		RefactorInstanceMember("Terraria.ModLoader.ModBlockType", "ItemDrop", Removed("No longer needed"));
```

Comment:
Consider `"Tiles and walls will drop the item which places them automatically. Use RegisterItemDrop to alter the automatic drop if necessary"`